lti_lib/student_score.js

```diff
@@ -0,0 +1,24 @@
+const axios = require('axios');
+
+function grading_grade(grade, payload) {
+
+    var headers = {
+        'Content-Type': 'application/json',
+    }
+    axios.post(payload["https://purl.imsglobal.org/spec/lti-ags/claim/endpoint"].lineitems, {
+        "userId": "3427",
+        "scoreGiven": grade,
+        "scoreMaximum": 1,
+        "comment": "Well done!",
+        "timestamp": "2018-06-12T14:30:23.605Z",
```

Comment:
you should add the actual time.  Looks like they want the JSON date:
`"timestamp": Date.now.toJSON(),`